Dockerfile

```diff
@@ -5,10 +5,14 @@ LABEL org.label-schema.name='AMOC Arsenal' \
 
 WORKDIR /amoc_arsenal
 
-COPY ./ ./
-RUN git clean -ffxd
+COPY rebar.lock .
```

Comment:
Ooooo true 😄

Reply:
Hmm but still. First I want to copy _only_ the rebar.lock so that I can create a docker layer with the dependencies, that one is intentional. Then I want to copy only the dirs declared, setting a dockerignore would not ignore if for example I created a new temporary file 🤔